legions/commands/commands.py

```diff
@@ -82,7 +82,9 @@ def getnodeinfo():
             cprint("Coinbase Account: {}".format(w3.eth.coinbase), "green") 
         except Exception as e:
             cprint("Coinbase not available: {}".format(e), "red")   
-            
+        cprint("Accounts", "green")
+        for account in w3.eth.accounts:
+            cprint("- {}".format(account), "green")
         cprint("Accounts: {}".format(w3.eth.accounts), "green")    
```

Comment:
Thanks for the update. Please remove the extra line printing all the accounts and I'll merge this.

Reply:
Thanks, done!
I will check the others issues too :)

JM